README.md

```diff
@@ -29,7 +29,7 @@
 
 # 📖 Movie App <a name="about-project"></a>
 
-**Movie App**  is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux. The application will allow users to search for appropriate choice movie. Application is working with the real live data from API.
+**Movie App**  is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux. The application will allow users to search for appropriate choice movie to view. Application is working with the real live data from API.
```

Comment:
- [ ] Please remove this white background on the home page, the user can't see the title clearly.
![image](https://user-images.githubusercontent.com/74744787/212478429-e3f322d2-0a4f-4b01-89c6-6ae39c9535c6.png)

- [ Optional ] you can create a pink hovering on the design pics this will make the pages match the design more.